app/Http/Controllers/NatureOfWorkController.php

```
@@ -0,0 +1,53 @@
+            'message' => 'Nature of Work created successfully',
+            'data' => $natureOfWork,
+        ], 201);
+    }
+    public function update(UpdateNatureOfWorkRequest $request, NatureOfWork $nature_of_work): JsonResponse
+    {
+        $nature_of_work->update($request->validated());
+        return response()->json([
+            'message' => 'Nature of Work updated successfully',
+            'data' => $nature_of_work,
+        ]);
+    }
```

PHPMD (2.15.0)

The parameter $nature_of_work is not named in camelCase. (Controversial Rules)

(CamelCaseParameterName)